packages/interface/package.json

```diff
@@ -0,0 +1,37 @@
+{
+	"name": "@wordpress/interface",
+	"version": "0.0.1",
+	"private": true,
+	"description": "Interface module for WordPress.",
```

Comment:
Do you have some ideas on making this description more friendly for those using npm to find packages?

Reply:
I extended a little bit the description, but for now, this package does not offer much and will probably not be very useful to third-party users.
When the functionality of this package is expanded and we are confident with what the package offers I guess we can expand even more the description and make it more concrete.